### DISCUSSION TITLE
Why isn't the responsive breakpoint taking precedence on text-color?

### DISCUSSION THREAD

**Discussion by char-cole** [tailwindlabs/tailwindcss#11028] [Help]
On a tablet device (width 820px), why is `md` color being overwritten by the default? Other styles are working fine, but text color is not:
<img width="262" alt="image" src="https://user-images.githubusercontent.com/38542275/232825775-f021c337-27b0-435a-ad06-f9f1ba87bba4.png">

**Comment by wongjn** [comment]
Consider providing a remotely-hosted git repository that reproduces the issue so that we may be able to help troubleshoot.

**Reply by wongjn** [reply]
> @wongjn of course, sorry for my haste. I found the problem in my code, so I'll explain here. is this expected behavior?
> 
> * this is a react component
> * I'm applying classes with `classNames`
> * some classes are static (such as text-blue)
> * other classes are returned from a ternary
> * classes from the ternary take lower precedence than static classes even if they're more specific to the breakpoint
> 
> ie: `className={classNames('text-blue', someVar ? 'md:text-black' : 'some-other-classes')}`
> 
> in the above, the text is blue at all breakpoints[^1]

This isn't the expected behavior. Consider providing a remotely-hosted git repository that reproduces the issue so that we may be able to help troubleshoot.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/11028#discussioncomment-5652820

**Comment by char-cole** [comment]
@wongjn of course, sorry for my haste. I found the problem in my code, so I'll explain here. is this expected behavior?

- this is a react component
- I'm applying classes with `classNames`
- some classes are static (such as text-blue)
- other classes are returned from a ternary
- classes from the ternary take lower precedence than static classes even if they're more specific to the breakpoint

ie:
`className={classNames('text-blue', someVar ? 'md:text-black' : 'some-other-classes')}`

in the above, the text is blue at all breakpoints

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/11028#discussioncomment-5652842 to keep the reply chain together.
